packages/database/src/migrations/1745283833905-addPatientProgramRegistrationDeletedAt.ts

```diff
@@ -0,0 +1,20 @@
+import { QueryInterface } from 'sequelize';
+
+export async function up(query: QueryInterface): Promise<void> {
+  // Set deletedAt for historical records with registrationStatus = recordedInError
+  // Use the date field value as deletedAt value, with proper timestamp conversion
+  await query.sequelize.query(`
+    UPDATE patient_program_registrations
+    SET deleted_at = date::timestamp
+    WHERE registration_status = 'recordedInError' AND deleted_at IS NULL
+  `);
+}
+
+export async function down(query: QueryInterface): Promise<void> {
+  // Revert the changes by setting deletedAt back to null
+  await query.sequelize.query(`
+    UPDATE patient_program_registrations
+    SET deleted_at = NULL
+    WHERE registration_status = 'recordedInError'
```

Comment:
```suggestion
    WHERE registration_status = 'recordedInError' and deleted_at = date::timestamp
```
? Very optional as we hardly run these and tbh I'm not sure if it's much more accurate

Reply:
I'm happy to include this because: yeah, we hardly run these, but also it's a least a smaller subset and not ALL recorded in error ones